include/modem/at_custom_cmd.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef AT_CUSTOM_CMD_H_
+#define AT_CUSTOM_CMD_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file at_custom_cmd.h
+ *
+ * @defgroup at_custom_cmd Custom AT commands
+ *
+ * @{
+ *
+ * @brief Public API for adding custom AT commands using filters in the Modem library with
+ * application callbacks.
+ *
+ */
+
+#include <stddef.h>
+#include <kernel.h>
+#include <sys/util_macro.h>
+#include <toolchain/common.h>
+#include <nrf_modem_at.h>
+
+/**
+ * @brief Initialize custom AT commands.
+ */
+int at_custom_cmd_init(void);
+
+/**
+ * @brief Deinitialize custom AT commands.
+ */
+int at_custom_cmd_deinit(void);
+
+/**
+ * @brief Fill response buffer without overflowing the buffer.
+ *
+ * @note For the modem library to accept the response in @c buf as a success, the response must
+ *       contain "OK\r\n". If the response is an error, use "ERROR\r\n" or appropriate CMS/CME
```

Comment:
```suggestion
 *       contain "OK\\r\\n". If the response is an error, use "ERROR\\r\\n" or appropriate CMS/CME
```
This will fix the doc build error. We have to escape the special characters.

Reply:
Thanks :)